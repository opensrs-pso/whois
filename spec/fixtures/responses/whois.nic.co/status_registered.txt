Domain Name:                                 T.CO
Domain ID:                                   D740225-CO
Sponsoring Registrar:                        MELBOURNE IT LTD
Sponsoring Registrar IANA ID:                13
Registrar URL (registration services):       www.melbourneit.com.au
Domain Status:                               serverDeleteProhibited
Domain Status:                               serverTransferProhibited
Registrant ID:                               TWITTERREG2012
Registrant Name:                             Twitter, Inc.
Registrant Organization:                     Twitter, Inc.
Registrant Address1:                         1355 Market Street
Registrant Address2:                         Suite 900
Registrant City:                             San Francisco
Registrant State/Province:                   CA
Registrant Postal Code:                      94103
Registrant Country:                          United States
Registrant Country Code:                     US
Registrant Phone Number:                     +1.4152229670
Registrant Facsimile Number:                 +1.4152220922
Registrant Email:                            domains@twitter.com
Administrative Contact ID:                   TWITTERADMIN2012
Administrative Contact Name:                 Domain Admin
Administrative Contact Organization:         Twitter, Inc.
Administrative Contact Address1:             1355 Market Street
Administrative Contact Address2:             Suite 900
Administrative Contact City:                 San Francisco
Administrative Contact State/Province:       CA
Administrative Contact Postal Code:          94103
Administrative Contact Country:              United States
Administrative Contact Country Code:         US
Administrative Contact Phone Number:         +1.4152229670
Administrative Contact Facsimile Number:     +1.4152220922
Administrative Contact Email:                domains@twitter.com
Billing Contact ID:                          MIT_BILLING
Billing Contact Name:                        DBS Billing
Billing Contact Address1:                    636 Ellis Street
Billing Contact City:                        Mountain View
Billing Contact State/Province:              CA
Billing Contact Postal Code:                 94043
Billing Contact Country:                     United States
Billing Contact Country Code:                US
Billing Contact Phone Number:                +1.8669073267
Billing Contact Facsimile Number:            +1.6506182574
Billing Contact Email:                       billing@melbourneitdbs.com
Technical Contact ID:                        TWITTERTECH2012
Technical Contact Name:                      Tech Admin
Technical Contact Organization:              Twitter, Inc.
Technical Contact Address1:                  1355 Market Street
Technical Contact Address2:                  Suite 900
Technical Contact City:                      San Francisco
Technical Contact State/Province:            CA
Technical Contact Postal Code:               94103
Technical Contact Country:                   United States
Technical Contact Country Code:              US
Technical Contact Phone Number:              +1.4152229670
Technical Contact Facsimile Number:          +1.4152220922
Technical Contact Email:                     domains-tech@twitter.com
Name Server:                                 NS1.P34.DYNECT.NET
Name Server:                                 NS2.P34.DYNECT.NET
Name Server:                                 NS3.P34.DYNECT.NET
Name Server:                                 NS4.P34.DYNECT.NET
Created by Registrar:                        INJECTCSR
Last Updated by Registrar:                   MELBOURNE IT LTD
Last Transferred Date:                       Mon Mar 14 22:21:58 GMT 2011
Domain Registration Date:                    Mon Apr 26 07:50:40 GMT 2010
Domain Expiration Date:                      Sat Apr 25 23:59:59 GMT 2015
Domain Last Updated Date:                    Tue Apr 16 13:53:26 GMT 2013

>>>> Whois database was last updated on: Tue May 07 12:18:16 GMT 2013 <<<<
.CO Internet, S.A.S., the Administrator for .CO, has collected this
information for the WHOIS database through Accredited Registrars.  
This information is provided to you for informational purposes only 
and is designed to assist persons in determining contents of a domain 
name registration record in the .CO Internet registry database. .CO 
Internet makes this information available to you "as is" and does not 
guarantee its accuracy.
 
By submitting a WHOIS query, you agree that you will use this data 
only for lawful purposes and that, under no circumstances will you 
use this data:  (1) to allow, enable, or otherwise support the transmission 
of mass unsolicited, commercial advertising or solicitations via direct 
mail, electronic mail, or by telephone; (2) in contravention of any 
applicable data and privacy protection laws; or (3) to enable high volume, 
automated,  electronic processes that apply to the registry (or its systems). 
Compilation, repackaging, dissemination, or other use of the WHOIS 
database in its entirety, or of a substantial portion thereof, is not allowed 
without .CO Internet's prior written permission. .CO Internet reserves the 
right to modify or change these conditions at any time without prior or 
subsequent notification of any kind. By executing this query, in any manner 
whatsoever, you agree to abide by these terms.  In some limited cases, 
domains that might appear as available in whois might not actually be 
available as they could be already registered and the whois not yet updated 
and/or they could be part of the Restricted list. In this cases, performing a 
check through your Registrar's (EPP check) will give you the actual status 
of the domain. Additionally, domains currently or previously used as 
extensions in 3rd level domains will not be available for registration in the 
2nd level. For example, org.co,mil.co,edu.co,com.co,net.co,nom.co,arts.co,
firm.co,info.co,int.co,web.co,rec.co,co.co. 
 
NOTE: FAILURE TO LOCATE A RECORD IN THE WHOIS DATABASE IS NOT 
INDICATIVE OF THE AVAILABILITY OF A DOMAIN NAME.   
 
All domain names are subject to certain additional domain name registration 
rules. For details, please visit our site at www.cointernet.co <http://www.cointernet.co>.
