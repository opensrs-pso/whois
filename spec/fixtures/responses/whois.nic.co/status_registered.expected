#domain
  should: %s == "t.co"

#domain_id
  should: %s == "D740225-CO"


#status
  should: %s == ["serverDeleteProhibited", "serverTransferProhibited"]

#available?
  should: %s == false

#registered?
  should: %s == true


#created_on
  should: %s CLASS(time)
  should: %s == Time.parse("2010-04-26 07:50:40 GMT")

#updated_on
  should: %s CLASS(time)
  should: %s == Time.parse("2013-04-16 13:53:26 GMT")

#expires_on
  should: %s CLASS(time)
  should: %s == Time.parse("2015-04-25 23:59:59 GMT")


#registrar
  should: %s CLASS(registrar)
  should: %s.id           == "13"
  should: %s.name         == "MELBOURNE IT LTD"

#registrant_contacts
  should: %s CLASS(array)
  should: %s SIZE(1)
  should: %s[0] CLASS(contact)
  should: %s[0].type          == Whois::Record::Contact::TYPE_REGISTRANT
  should: %s[0].id            == "TWITTERREG2012"
  should: %s[0].name          == "Twitter, Inc."
  should: %s[0].organization  == "Twitter, Inc."
  should: %s[0].address       == "1355 Market Street\nSuite 900"
  should: %s[0].city          == "San Francisco"
  should: %s[0].zip           == "94103"
  should: %s[0].state         == "CA"
  should: %s[0].country       == "United States"
  should: %s[0].country_code  == "US"
  should: %s[0].phone         == "+1.4152229670"
  should: %s[0].fax           == "+1.4152220922"
  should: %s[0].email         == "domains@twitter.com"

#admin_contacts
  should: %s CLASS(array)
  should: %s SIZE(1)
  should: %s[0] CLASS(contact)
  should: %s[0].type          == Whois::Record::Contact::TYPE_ADMIN
  should: %s[0].id            == "TWITTERADMIN2012"
  should: %s[0].name          == "Domain Admin"
  should: %s[0].organization  == "Twitter, Inc."
  should: %s[0].address       == "1355 Market Street\nSuite 900"
  should: %s[0].city          == "San Francisco"
  should: %s[0].zip           == "94103"
  should: %s[0].state         == "CA"
  should: %s[0].country       == "United States"
  should: %s[0].country_code  == "US"
  should: %s[0].phone         == "+1.4152229670"
  should: %s[0].fax           == "+1.4152220922"
  should: %s[0].email         == "domains@twitter.com"

#technical_contacts
  should: %s CLASS(array)
  should: %s SIZE(1)
  should: %s[0] CLASS(contact)
  should: %s[0].type          == Whois::Record::Contact::TYPE_TECHNICAL
  should: %s[0].id            == "TWITTERTECH2012"
  should: %s[0].name          == "Tech Admin"
  should: %s[0].organization  == "Twitter, Inc."
  should: %s[0].address       == "1355 Market Street\nSuite 900"
  should: %s[0].city          == "San Francisco"
  should: %s[0].zip           == "94103"
  should: %s[0].state         == "CA"
  should: %s[0].country       == "United States"
  should: %s[0].country_code  == "US"
  should: %s[0].phone         == "+1.4152229670"
  should: %s[0].fax           == "+1.4152220922"
  should: %s[0].email         == "domains-tech@twitter.com"


#nameservers
  should: %s CLASS(array)
  should: %s SIZE(4)
  should: %s[0] CLASS(nameserver)
  should: %s[0].name == "ns1.p34.dynect.net"
  should: %s[1] CLASS(nameserver)
  should: %s[1].name == "ns2.p34.dynect.net"
  should: %s[2] CLASS(nameserver)
  should: %s[2].name == "ns3.p34.dynect.net"
  should: %s[3] CLASS(nameserver)
  should: %s[3].name == "ns4.p34.dynect.net"
